ssm/README.md

```diff
@@ -0,0 +1,30 @@
+# ssm
+
+A Wing library for working with [Systems Manager](https://aws.amazon.com/systems-manager/).
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/ssm
+```
+
+## Usage
+
+```js
+bring expect;
+bring ssm;
+
+let param = new ssm.Parameter(name: "/test/param1", value: "value1");
```

Comment:
Kind of feels like we should support the use case of just referencing an existing parameter.

I am not sure I fully understand the use case for defining the parameter value in the same Wing app that references it (what's the benefit of going through SSM here?).

```js
let bla = new ssm.ParameterReference(name: "bla");

inflight () => {
  log(bla.value());
};
```

Reply:
But it seems like the value is a required prop of `ssm.Parameter`...

Reply:
Anyway - I think we should support both use cases:

```js
bring ssm;

// with value
new ssm.Parameter(name: "my-param", value: "my-value");

// without value
new ssm.ParameterReference(name: "my-param");
```

Maybe both can implement `IParameter`:

```js
interface IParameter {
  inflight value(): str;
}
```

Or something like this.